task1/src/app/cource-item-date.directive.ts

```diff
@@ -0,0 +1,25 @@
+import {Directive, ElementRef, Input, OnInit} from '@angular/core';
+ 
+@Directive({
+    selector: '[fresh]'
+})
+export class CourceItemDateDirective{
+    @Input() courceDate = "";
+        
+    ngOnInit(){
+        const twoWeeksAgo= (date) => {
+            const twoWeeks = 14 * 24 * 1000 * 60 * 60;
+            const twoWeeksAgo = Date.now() - twoWeeks;
+        
+            return date > twoWeeksAgo;
+        }
```

Comment:
mozesz przeniesc deklaracje funkcji poza ngOnInit,
argument wejsciowy jest zbedny, uzyj this.courseDate